### ISSUE TITLE
Data Bridge MQTT Sink Payload can't be empty

### ISSUE THREAD

**Issue by Tuxzx** [emqx/emqx#8768]
### What happened?

when i set payload empty, I got this.

<img width="1080" alt="Screen Shot 2022-08-21 at 1 07 24 AM" src="https://user-images.githubusercontent.com/29911719/185759021-8530764b-03bc-4b17-8b43-8904a7d199eb.png">

### What did you expect to happen?

get the raw event message.

### How can we reproduce it (as minimally and precisely as possible)?

start emqx 5.0.5, create data bridge which mqtt sink and set payload empty.

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.5
datetime  : 2022-08-20T17:22:06.322395447+00:00
uptime    : 30 minutes, 57 seconds
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
$ uname -a
Linux 07c13c12aecf 5.10.104-linuxkit #1 SMP Thu Mar 17 17:08:06 UTC 2022 x86_64 GNU/Linux
```

</details>

### Log files

<details>
2022-08-20T17:24:09.113641+00:00 [warning] entrypoint: <<"emqx:update_config/3">>, kind: initiate, line: 492, mfa: emqx_cluster_rpc:log_and_alarm/3, msg: cluster_rpc_apply_result, result: {error,{#{roots => [{bridges,{ref,emqx_bridge_schema,bridges}}],translations => #{},validations => [{check_process_watermark,#Fun<emqx_schema.7.35326886>},{check_cpu_watermark,#Fun<emqx_schema.8.35326886>},{check_ssl_opts,#Fun<emqx_authz_schema.3.112820419>}]},[#{kind => validation_error,mismatches => #{<<"egress">> => #{kind => validation_error,path => "bridges.mqtt.fasdfsadf.payload",reason => mandatory_required_field,value => undefined},<<"ingress">> => #{kind => validation_error,path => "bridges.mqtt.fasdfsadf.direction",reason => #{expected_type => <<"ingress">>},value => "egress"}},path => "bridges.mqtt.fasdfsadf",reason => matched_no_union_member}]}}, tnx_id: 22
</details>

**Comment by Tuxzx** [comment]
And another.
When i created a bridge successfully, back to bridge list and click it, it disappear.

This birdge create with default `${payload}` payload.
<img width="1080" alt="Screen Shot 2022-08-21 at 1 28 52 AM" src="https://user-images.githubusercontent.com/29911719/185759278-b8c40b5a-e0c6-4d52-ab0c-61a55bd9905c.png">

You can see the url, I'm in the bridge page.
<img width="1080" alt="Screen Shot 2022-08-21 at 1 28 55 AM" src="https://user-images.githubusercontent.com/29911719/185759309-6fd90a97-4491-4733-93bf-82d5ebc49f32.png">

**Comment by terry-xiaoyu** [comment]
Will be fixed in the next release.

**Comment by terry-xiaoyu** [comment]
@Tuxzx Do you really send an `empty` payload or send all the available fields (in json format) to the remote MQTT broker?

**Comment by Tuxzx** [comment]
> @Tuxzx Do you want to send an `empty` payload or send all the available fields (in json format) to the remote MQTT broker?

I've followed this Description, I want to send all the available fields to the remote broker.
<img width="472" alt="Screen Shot 2022-08-23 at 2 51 47 PM" src="https://user-images.githubusercontent.com/29911719/186092247-98ea50d9-deac-4548-9edc-faa8acf13380.png">


In version 5.0.4, I can create a data bridge with the default payload `${payload}`, when the data bridge creates success, I can update the payload to empty.

But In version 5.0.5, I just can see the data bridge in the list, when I click, it disappears. https://github.com/emqx/emqx/issues/8768#issuecomment-1221372739

**Comment by Tuxzx** [comment]
Fixed at 5.0.8
